### ISSUE TITLE
Broken permalink to https://w3id.org/ontouml-models/catalog on Wiki

### ISSUE THREAD

**Issue by SPowley** [OntoUML/ontouml-models#276]
Hi, the link to https://scs-ontouml.eemcs.utwente.nl/catalog is broken on the OntoUML Wiki page https://github.com/OntoUML/ontouml-models#catalog-structure

The error is in list item 1 which reads:

catalog.ttl: is the file encoding the catalog itself in a triple-based format - i.e., the aggregated data of all ontologies that are part of the catalog - its permanent identifier is https://w3id.org/ontouml-models/catalog;

**Comment by pedropaulofb** [comment]
@SPowley, thank you very much for reporting this issue! 

The catalog recently passed through a FAIRification process, where we significantly improved it. However, by doing that, the catalog's GitHub documentation got outdated. I undertake to correct this documentation as soon as possible.

You can access the catalog's FAIR Data Point using the following link: https://w3id.org/ontouml-models
Also, you can access the single file containing all catalog data using the following link: 

I kindly ask you to send me an email ([pfavatobarcelos@unibz.it](mailto:pfavatobarcelos@unibz.it)) so I can send you the most recent documentation we have.

**Comment by pedropaulofb** [comment]
README.md updated. All broken links were fixed. Thank you once again, @SPowley .

**Comment by SPowley** [comment]
> README.md updated. All broken links were fixed. Thank you once again, @SPowley .

Seems to be a syntax error in the link under

> **Catalog Releases**
> The catalog also offers releases comprising all its data and metadata compiled into a single Turtle file. Releases are tagged after the following nomenclature <YYYY><MM><DD> and can be directly accessed via the permanent identifier [https://w3id.org/ontouml-models/release/*\](https://w3id.org/ontouml-models/release/*%5C)<release_tag>*.

Clicking the link lands on a 404

**Comment by pedropaulofb** [comment]
Yes, you're right! Thank you for noticing @SPowley ! I already fixed it. 

Please let me know if you find something else.
